.vscode/settings.json

```
@@ -7,5 +7,5 @@
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
   "files.trimTrailingWhitespace": true,
-  "search.usePCRE2": true
+  "search.usePCRE2": true,
```

```suggestion
  "search.usePCRE2": true
```
I guess VS Code is OK with it, but strictly speaking JSON files shouldn't have trailing commas.